### ISSUE TITLE
curlpp-config doesn't work

### ISSUE THREAD

**Issue by equal-l2** [Homebrew/homebrew-core#9315]
- [x] Ran `brew update` and retried your prior step?
- [x] Ran `brew doctor`, fixed as many issues as possible and retried your prior step?
- [x] Confirmed this is problem with specific formulae and not Homebrew/brew? If it's a general Homebrew/brew problem please file this issue at https://github.com/Homebrew/brew/issues/new

### Bug reports:
`curlpp-config` , in formula `curlpp`, doesn't work.

Output:
```
/usr/local/bin/curlpp-config: /bin/sh^M: bad interpreter: No such file or directory
```

### Brew status:
`brew config`
```
HOMEBREW_VERSION: 1.1.8-46-g78d584f42f
ORIGIN: https://github.com/Homebrew/brew.git
HEAD: 78d584f42f2fa07f4c5f60c1aabd5de2e8fc75c3
Last commit: 8 hours ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 5e19cf1248328d8e99e1b39311e6c5ce6170b782
Core tap last commit: 2 hours ago
HOMEBREW_PREFIX: /usr/local
HOMEBREW_REPOSITORY: /usr/local/Homebrew
HOMEBREW_CELLAR: /usr/local/Cellar
HOMEBREW_BOTTLE_DOMAIN: https://homebrew.bintray.com
CPU: quad-core 64-bit ivybridge
Homebrew Ruby: 2.0.0-p648
Clang: 8.0 build 800
Git: 2.11.0 => /usr/local/bin/git
Perl: /usr/bin/perl
Python: /usr/bin/python
Ruby: /usr/bin/ruby => /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/bin/ruby
Java: 1.8.0_112, 1.7.0_71
macOS: 10.12.3-x86_64
Xcode: 8.2.1
CLT: 8.2.0.0.1.1480973914
X11: 2.7.11 => /opt/X11
```

`brew doctor`
```
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry and just ignore them. Thanks!

Warning: Unbrewed header files were found in /usr/local/include.
If you didn't put them there on purpose they could cause problems when
building Homebrew formulae, and may need to be deleted.

Unexpected header files:
  /usr/local/include/python3.6m/greenlet/greenlet.h
```

**Comment by DomT4** [comment]
It would seem the executable is CRLF rather than just LF for some reason. Perhaps the simplest fix for you is to either have an editor fix that for you, or `brew install dos2unix` followed by:

```bash
dos2unix /usr/local/Cellar/curlpp/0.7.4/bin/curlpp-config
```
Not sure if this was a conscious choice by upstream or not.

**Comment by equal-l2** [comment]
Well, I found homebrew is not responsible of this problem.
This problem occurs with manual build.

Thanks, @DomT4!

**Comment by equal-l2** [comment]
I've opened a issue on upstream.
jpbarrette/curlpp#34

**Comment by equal-l2** [comment]
Now closing since fixed on #9326
